hpobench/util/fairness_metrics.py

```diff
@@ -0,0 +1,96 @@
+"""
+This file contains functionality to compute various fairness related risk scores.
+"""
+
+import numpy as np
+
+STATISTICAL_DISPARITY = 'statistical_disparity'  # P(1 | group A) - P(1 | group B)
+UNEQUAL_OPPORTUNITY = 'unequal_opportunity'  # P(1 | group A, 0) - P(1 | group B, 0)
+UNEQUALIZED_ODDS = 'unequalized_odds'  # P(1 | group A, 1) - P(1 | group B, 1)
+
+TPR0 = 'tpr0'
+TPR1 = 'tpr1'
+TPR_DIF = 'tpr_dif'
+TPR_MIN = 'tpr_min'
+
+FAIRNESS_METRICS = [STATISTICAL_DISPARITY, UNEQUAL_OPPORTUNITY, UNEQUALIZED_ODDS, TPR0, TPR1, TPR_DIF, TPR_MIN]
+
+PRED_THRESHOLD = 0.5
+
+
+def fairness_risk(x, y, sensitive_rows, model, unfairness_metric):
```

Comment:
Are you sure you do not wanna just use the metrics from `fairlearn`? Or is that the original code?

Reply:
Hmmm… perhaps keep the original code, then we are closer to the original work

Reply:
If there is original code, would it be possible to import this code? Or at least leave a comment saying where the original code is?